src/lib/inspector.js

```diff
@@ -179,15 +179,17 @@ Inspector.prototype = {
 
     Events.on('domModified', (mutations) => {
       var self = this;
-      mutations.forEach((mutation) => {
-        if (mutation.type === 'childList') {
-          Array.prototype.slice.call(mutation.removedNodes).forEach(function (removedNode) {
-            if (self.selectedEntity === removedNode) {
-              self.selectEntity(null);
-            }
-          });
-        }
-      });
+      if (mutations) {
```

Comment:
If you return early, you could reduce the indent of the block below

`if (!mutations) { return; }`

`if (mutation.type !=='childList') { return; }`